spec/rubocop/ast/range_node_spec.rb

```diff
@@ -31,5 +31,15 @@
       it { expect(range_node.is_a?(described_class)).to be(true) }
       it { expect(range_node.range_type?).to be(true) }
     end
+
+    context 'with a beiginless range' do
```

Comment:
"beiginless" -> "beginless" 🙂

Reply:
Oops 💦 I fixed this typo by https://github.com/rubocop-hq/rubocop/commit/3e1bf9db8f1fac959b246cb4d0840a40e14744bf. Thanks!